kernel/configs/arm64_kata_kvm_5.4.x

```diff
@@ -1,6 +1,6 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/arm64 5.4.3 Kernel Configuration
+# Linux/arm64 5.4.15 Kernel Configuration
```

Comment:
Nice to have the update! - ooi, have you done any [metrics comparisons](https://github.com/kata-containers/tests/tree/master/metrics/report) between the old and new config?

Reply:
yep, I'll do the metrics comparisons asap. ;)